doc/tutorials/snippet-sharing-rest-service.md

```diff
@@ -0,0 +1,196 @@
+# REST Service for Sharing of Configuration Snippets #
+
+This tutorial explains how to install, configure and run everything that is required
+for the Snippet Sharing service and website. As operating system we expect Debian
+Jessie to be used, although most parts should also be applicable to other systems.
+
+## What is this? ##
+
+The Snippet Sharing service consists of two parts, a backend (the REST server) and
+a frontend (an AngularJS application executed in the browser), which does also house
+the Elektra website. The service itself can be used to share configuration snippets,
+i.e. configuration files of arbitrary applications.
```

Comment:
If it fits into the tutorial then you can add it there. Otherwise you can directly start with a new file in doc/news/. Btw. did you also see the news in doc/old?